### DISCUSSION TITLE
首页的卡片流是否可以显示一个 logo 用于标识`充电专属`的视频

### DISCUSSION THREAD

**Discussion by cuiko** [BewlyBewly/BewlyBewly#751] [Ideas]
<details>
<summary>现在的样式</summary>
![now](https://github.com/BewlyBewly/BewlyBewly/assets/33982919/c15d2677-0b05-4681-a4c6-96f14ef18725)
</details>
<details>
<summary>可参考的样式</summary>
![reference](https://github.com/BewlyBewly/BewlyBewly/assets/33982919/f9de8d05-3b20-4210-b639-bd88a141cb93)
</details>
<details>
<summary>数据支撑</summary>
![data](https://github.com/BewlyBewly/BewlyBewly/assets/33982919/276410f0-b891-4d27-987c-5b31f1dabe82)
</details>

**Comment by cuiko** [comment]
> 现在的样式
> ![now](https://github.com/BewlyBewly/BewlyBewly/assets/33982919/c15d2677-0b05-4681-a4c6-96f14ef18725)
> 可参考的样式
> ![reference](https://github.com/BewlyBewly/BewlyBewly/assets/33982919/f9de8d05-3b20-4210-b639-bd88a141cb93)
> 数据支撑
> ![data](https://github.com/BewlyBewly/BewlyBewly/assets/33982919/276410f0-b891-4d27-987c-5b31f1dabe82)
怎么显示不了图片。。。

**Comment by hakadao** [comment]
現在版本應該實現了
